azure-pipelines.yml

```diff
@@ -14,6 +14,8 @@ pr:
 
 variables:
 - template: ${{ variables.Pipeline.Workspace }}/.azure-pipelines/templates/variables.yml
+- name: Codeql.Enabled
+  value: false
```

Comment:
I have to use this syntax instead of `Codeql.Enabled: false`, because we use varable `template`.

Ref: https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#specify-variables